.agents/skills/deploy/SKILL.md

```
@@ -0,0 +1,22 @@
+---
+name: deploy
+description: Build and deploy openperouter to a kind cluster, optionally in hostmode
+trigger: deploy, deploy hostmode, ship it, deploy the project
+---
+
+# Deploy openperouter
+
+Run these steps in order:
+
+1. Build the docker image:
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)